### PR TITLE
add zh_cn translation for error

### PR DESCRIPTION
**Pull request by WangMengabc**

### CODE REVIEW

**Comment by wenningerk** [comment]
test this please

File: po/zh_CN.po [hunk 1]
```diff
@@ -719,16 +715,46 @@ msgid ""
...
+msgstr "需要使用-t指定资源类型"
+
+#: tools/crm_resource.c:1155
+#, c-format
+msgid "No agents found for standard '%s'"
+msgstr "没有发现标准代理'%s'"
+
+#: tools/crm_resource.c:1389
+msgid "Could not create executor connection"
+msgstr "无法创建资源程序链接"
```
Comment (gao-yan) [inline po/zh_CN.po:729]: The term "executor" is translated/interpreted to "pacemaker-execd 守护进程" in other places:

https://github.com/ClusterLabs/pacemaker/commit/27a77334f4#diff-0a4a9ca503c41c5f2a54de36293d8863d6ef72d42476226fa0b9eddf52bc94e9R683

Please keep it consistent.

So this basically means "无法创建到 pacemaker-execd 守护进程的连接".

File: po/zh_CN.po [hunk 2]
```diff
@@ -719,16 +715,46 @@ msgid ""
...
+msgid "Must supply a resource id with -r"
+msgstr "必须使用-r指定资源id"
+
+#: tools/crm_resource.c:1717
+msgid "Must supply a node name with -N"
+msgstr "必须使用-N指定节点名称"
+
+#: tools/crm_resource.c:1741
+msgid "Could not create CIB connection"
+msgstr "不能创建CIB连接"
```
Comment (gao-yan) [inline po/zh_CN.po:756]: Should be better to be consistent with the above "Could not" case with something like "无法创建到 CIB 的连接".

**Review by gao-yan — CHANGES_REQUESTED** [review]

**Comment by kgaillot** [comment]
Hi,

Since these are command-line output messages, and not log messages, I'm OK with translating them, but I recommend thinking about whether it is worth it. Unlike the option descriptions, these messages are more likely to change from release to release, so it will require ongoing effort to keep them up to date. With the small number of messages here, that wouldn't be significant, but if you plan on eventually translating all of them, it likely will be.

If you do want to proceed, please squash these commits into one, and rebase on current main to avoid the merge commit. Thanks!

**Comment by kgaillot** [comment]
ok to test

**Comment by wenningerk** [comment]
Still looking as if you have multiple commits on your branch.
A little bit unfortunate that you were working on the main.

I personally keep the official branches as is, both on my
clone on github as well as my local clones. That makes it
easy to revert to main or pull current changes from upstream
into main.

If a local branch is messed up too much I usually stop messing
around with rebase (incl. squash) & reset but rather try to move
the changes cleanly into a new branch (squashing them before
or after depends on the situation and/or personal preferences)
and force push that to my work-branch on github.
Only bad impact of force-pushing to the branch the PR was
created for is that the discussion may look a bit messed up as
the commits referred to may be gone. Guess this is the reason
why some people prefer keeping the branch as is, close the
PR and create a new one for a new branch. In my eyes that
is just another kind of (depending on the volume of the discussion
supposedly bigger) mess.

**Comment by wenningerk** [comment]
aah cool ... merge commits gone.
as I had understood Ken meant to squash the 2 code-commits on top.

File: po/zh_CN.po [hunk 3]
```diff
@@ -719,16 +715,183 @@ msgid ""
...
+msgstr "必须使用-N指定节点名称"
+
+#: tools/crm_resource.c:1741
+msgid "Could not create CIB connection"
+msgstr "无法创建到CIB的连接"
+
+#: tools/crm_resource.c:1749
+#, fuzzy, c-format
+msgid "Could not connect to the CIB: %s"
+msgstr "%s不能连接到CIB"
```
Comment (gao-yan) [inline po/zh_CN.po:858]: Since `%s` is the error message, we could keep it like "不能连接到CIB： %s".

File: po/zh_CN.po [hunk 4]
```diff
@@ -719,16 +715,183 @@ msgid ""
...
+
+#: tools/crm_resource.c:1770
+#, c-format
+msgid "Resource '%s' not found"
+msgstr "没有发现'%s'资源"
+
+#: tools/crm_resource.c:1782
+#, c-format
+msgid "Cannot operate on clone resource instance '%s'"
+msgstr "不能操作克隆资源'%s'"
```
Comment (gao-yan) [inline po/zh_CN.po:868]: We should be more specific that this is about "克隆资源实例".

File: po/zh_CN.po [hunk 5]
```diff
@@ -719,16 +715,183 @@ msgid ""
...
+
+#: tools/crm_resource.c:2050
+#, fuzzy
+msgid "You need to supply a value with the -v option"
+msgstr "需要使用-v选项提供一个值"
+
+#: tools/crm_resource.c:2105
+#, c-format
+msgid "Unimplemented command: %d"
+msgstr "未生效的命令：%d"
```
Comment (gao-yan) [inline po/zh_CN.po:888]: Could be better with "无效的命令".

**Review by gao-yan — CHANGES_REQUESTED** [review]

File: po/zh_CN.po [hunk 6]
```diff
@@ -719,16 +715,183 @@ msgid ""
...
+
+#: tools/crm_resource.c:1597
+msgid "--expired requires --clear or -U"
+msgstr "--expired需要和--clear或-U一起使用"
+
+#: tools/crm_resource.c:1614
+#, c-format
+msgid "Error parsing '%s' as a name=value pair"
+msgstr "'%s'解析错误，解析为name=value"
+
```
Comment (gao-yan) [inline po/zh_CN.po:841]: Should be better with "'%s' 解析错误，格式为 name=value ".

**Review by gao-yan — APPROVED** [review]
Would be better to squash the additional commits for revisions into the previous.  The content looks good though..

**Pull request closed without merging**